tests/shakedown/shakedown/clients/mesos.py

```diff
@@ -1006,491 +951,6 @@ def __str__(self):
             return "master:{0}".format(self._path)
 
 
-class TaskIO(object):
```

Comment:
Why was this removed? I get that *we* don't use it _now_ but otherwise it might be useful?